src/transformers/models/trillsson_efficientnet/modeling_trillsson_efficientnet.py

```diff
@@ -0,0 +1,610 @@
+        self.dense = nn.Linear(output_channel, config.output_size)
+
+        # Initialize weights
+        self.post_init()
+
+    @add_start_docstrings_to_model_forward(TRILLSSON_EFFICIENTNET_INPUTS_DOCSTRING)
+    @add_code_sample_docstrings(
+        processor_class=_FEAT_EXTRACTOR_FOR_DOC,
+        checkpoint=_CHECKPOINT_FOR_DOC,
+        output_type=BaseModelOutputWithPoolingAndNoAttention,
+        config_class=_CONFIG_FOR_DOC,
+        modality="audio",
+        expected_output=_EXPECTED_OUTPUT_SHAPE,
+    )
+    def forward(
+        self,
+        input_values: Optional[torch.Tensor] = None,
+        output_hidden_states: Optional[bool] = None,
+        return_dict: Optional[bool] = None,
+    ) -> Union[tuple, BaseModelOutputWithPoolingAndNoAttention]:
+
+        output_hidden_states = (
+            output_hidden_states if output_hidden_states is not None else self.config.output_hidden_states
+        )
+        return_dict = return_dict if return_dict is not None else self.config.use_return_dict
+
+        if input_values is None:
+            raise ValueError("You have to specify input_values")
+        batch_size, _, frame_width, num_mel_bins = input_values.shape
+        input_values = torch.reshape(input_values, [-1, 1, frame_width, num_mel_bins])
+
+        all_hidden_states = () if output_hidden_states else None
+
+        hidden_states = self.stem(input_values)
+        for i, layer_module in enumerate(self.block):
+            hidden_states = layer_module(hidden_states)
+            if output_hidden_states:
+                all_hidden_states = all_hidden_states + (hidden_states,)
+        hidden_states = self.top(hidden_states)
+        if output_hidden_states:
+            all_hidden_states = all_hidden_states + (hidden_states,)
+
+        if self.pooler is not None:
+            pooled_output = self.pooler(hidden_states)
+            hidden_states = pooled_output
+        else:
+            pooled_output = None
+
+        hidden_states = hidden_states.view(hidden_states.size(0), -1)
+        main_net_output_dim = hidden_states.shape[-1]
+        hidden_states = hidden_states.reshape([batch_size, -1, main_net_output_dim])
+
+        hidden_states = torch.mean(hidden_states, dim=1, keepdim=False)
+        last_hidden_state = self.dense(hidden_states)
+        if not return_dict:
+            return tuple(v for v in [last_hidden_state, pooled_output, all_hidden_states] if v is not None)
+
+        return BaseModelOutputWithPoolingAndNoAttention(
+            last_hidden_state=last_hidden_state,
+            pooler_output=pooled_output,
+            hidden_states=all_hidden_states,
+        )
+
+
+@add_start_docstrings(
+    """
+    Trillsson_efficient Model with a sequence classification head on top (a linear layer over the last hidden state
+    output).
+    """,
+    TRILLSSON_EFFICIENTNET_START_DOCSTRING,
+)
+class TrillssonEfficientNetForSequenceClassification(TrillssonEfficientNetPreTrainedModel):
```

Comment:
usually sequence classification is for NLP models, we should rather have the `AudioClassification` WDYT?